mobile/src/main/res/values/dimens.xml

```diff
@@ -13,5 +13,5 @@
     <dimen name="outlineButtonStrokeWidth">2dp</dimen>
     <dimen name="widget_background_radius">8dp</dimen>
 
-    <integer name="section_switch_max_buttons">5</integer>
+    <integer name="section_switch_max_buttons">4</integer>
```

Comment:
At least on my device with 450dp width 5 regular buttons can be shown without any issue. Does it make sense to add resource files with `section_switch_max_buttons = [45678]`?

Reply:
Sure, why not? A regular phone is about 360dp wide, I think.